duckduckgo/macos-browser
# Disable testability for Release and Review builds

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203716819612150/f
CC: @brindy 

**Description**:
Fix a misconfiguration in the project-level settings.



**Steps to test this PR**:
1. Create a review and release builds (either with GHA or locally)
2. Verify that they’re ca. 50MB and not 110MB.

**Testing checklist**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
